jobboard/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from tinymce import models as tinymce_models
+
+
+class Company(models.Model):
+    name = models.CharField(max_length=200)
```

Comment:
Would it be worth having a slug field on the company which would enable a view which filtered the jobs by company?

Reply:
Done!